packages/amplify-provider-awscloudformation/src/push-resources.ts

```diff
@@ -555,6 +555,33 @@ export const updateStackForAPIMigration = async (context: $TSContext, category:
   }
 };
 
+/**
+ * Publish files that Amplify Studio depends on outside the zip file so that can read
+ * without streaming from the zip.
+ */
+const uploadStudioBackendFiles = async (s3: S3, bucketName: string) => {
+  const amplifyDirPath = pathManager.getAmplifyDirPath();
+  const studioBackendDirName = 'studio-backend';
+  // Delete the contents of the studio backend directory first
+  await s3.deleteDirectory(bucketName, studioBackendDirName);
+  // Create a list of file params to upload to the deployment bucket
+  const uploadFileParams = [
+    'cli.json',
+    'amplify-meta.json',
+    'backend-config.json',
+    'schema.graphql',
+    'transform.conf.json',
+    'parameters.json',
```

Comment:
what values in parameters.json does studio use?

Reply:
We only use it for the storage and auth categories, but there are params in there about permissions and 3p idps that we need.